src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/KeyFrameCriteria.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System;
 using System.Text;
+using System.Threading;
```

Comment:
Was this added accidentally? I don't see any change here that would need this import but maybe I am missing something.